kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/coordinator/transaction/TransactionMarkerChannelManager.java

```diff
@@ -329,7 +330,13 @@ public void addTxnMarkersToBrokerQueue(String transactionalId,
 
                         addressFuture.whenComplete((address, throwable) -> {
                             if (throwable != null) {
-                                log.warn("Failed to find broker for topic partition {}", topicPartition, throwable);
+                                if (throwable instanceof PulsarClientException.LookupException
+                                        || throwable.getCause() instanceof PulsarClientException.LookupException) {
+                                    log.warn("Failed to find broker for topic partition {} - {}", topicPartition,
+                                            throwable + "");
+                                } else {
+                                    log.warn("Failed to find broker for topic partition {}", topicPartition, throwable);
+                                }
```

Comment:
How could it reduce the spammy logs? For the lookup exception, there are still warn logs.

Reply:
Yes, but when the cause is `LookupException`, it won't print the stack info.

Reply:
>  it won't print the stack info.

I think not. See line 335, it's still `log.warn`.